### PR TITLE
replace old bools by new bools

### PR DESCRIPTION
**Pull request by aqua-uru**
This is excluding occurrences in strings.

### CODE REVIEW

File: Python/xTelescope.py [hunk 1]
```diff
@@ -89,9 +89,9 @@ def OnFirstUpdate(self):
     def Load(self):
         global boolScopeOperated
 
-        solo = true
+        solo = True
         if len(PtGetPlayerList()):
-            solo = false
+            solo = False
```
Comment (Hoikas) [inline Python/xTelescope.py:94]: This could be simplified to `solo = bool(PtGetPlayerList())`

File: Python/xTriggerNamedResponder.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def __init__(self):
         PtDebugPrint("__init__xTriggerNamedResponder v. %d" % self.version)
 
     def OnNotify(self,state,id,events):
-        if state != true:
+        if state != True:
```
Comment (Hoikas) [inline Python/xTriggerNamedResponder.py:73]: I think `if not state:` would be more Pythonic. Ideally we would do `if state is False:` but I think `state` is actually a float, so this needs to change either way.

File: Python/xLinkingBookGUIPopup.py [hunk 3]
```diff
@@ -149,7 +149,7 @@ def OnNotify(self,state,id,events):
         if id==shareRegion.id:
             if PtWasLocallyNotified(self.key):
                 for event in events:
-                    if (event[0]==kCollisionEvent and event[1] == false): # false being an exit
+                    if (event[0]==kCollisionEvent and event[1] == False): # False being an exit
```
Comment (Hoikas) [inline Python/xLinkingBookGUIPopup.py:152]: `event[1]` is an int - it would be better to `and not event[1]`

File: Python/xExitSubworld.py [hunk 4]
```diff
@@ -80,7 +80,7 @@ def OnNotify(self,state,id,events):
         for event in events:
             if (event[0] == kCollisionEvent):
                 entry = event[1]  # are we entering or exiting?
-                if (id == exitRgn.id and inSafetyRegion == false and entry == 0):
+                if (id == exitRgn.id and inSafetyRegion == False and entry == 0):
```
Comment (Hoikas) [inline Python/xExitSubworld.py:83]: `not inSafetyRegion`
`not entry`

File: Python/xAgeSDLBoolAndRespond.py [hunk 5]
```diff
@@ -129,10 +129,10 @@ def OnSDLNotify(self,VARname,SDLname,playerID,tag):
         PtDebugPrint("DEBUG: xAgeSDLBoolAndRespond.OnSDLNotify():\tnotification from playerID: %d" % (playerID))
 
         # does the change change our current state?
-        if boolCurrentState == false and (ageSDL[stringVar1Name.value][0] and ageSDL[stringVar2Name.value][0]):
-            boolCurrentState = true
-        elif boolCurrentState == true and not (ageSDL[stringVar1Name.value][0] and ageSDL[stringVar2Name.value][0]):
-            boolCurrentState = false
+        if boolCurrentState == False and (ageSDL[stringVar1Name.value][0] and ageSDL[stringVar2Name.value][0]):
```
Comment (Hoikas) [inline Python/xAgeSDLBoolAndRespond.py:132]: `if not boolCurrentState`

File: Python/xAgeSDLBoolAndRespond.py [hunk 6]
```diff
@@ -129,10 +129,10 @@ def OnSDLNotify(self,VARname,SDLname,playerID,tag):
...
+            boolCurrentState = True
+        elif boolCurrentState == True and not (ageSDL[stringVar1Name.value][0] and ageSDL[stringVar2Name.value][0]):
```
Comment (Hoikas) [inline Python/xAgeSDLBoolAndRespond.py:134]: `elif boolCurrentState`

File: Python/tldnWRCCBrain.py [hunk 7]
```diff
@@ -351,7 +351,7 @@ def OnSDLNotify(self,VARname,SDLname,playerID,tag):
             ###########
             if VARname == kStringAgeSDLBucketLoopMode:
                 bucketMode = ageSDL[kStringAgeSDLBucketLoopMode][0]
-                if bucketMode == true:
+                if bucketMode == True:
```
Comment (Hoikas) [inline Python/tldnWRCCBrain.py:354]: Implicity truthy.

File: Python/tldnWRCCBrain.py [hunk 8]
```diff
@@ -473,9 +473,9 @@ def OnNotify(self,state,id,events):
         if id == actBucketLoopMode.id:
             ageSDL = PtGetAgeSDL()
             bucketMode = ageSDL[kStringAgeSDLBucketLoopMode][0]
-            if bucketMode == true:
-                ageSDL[kStringAgeSDLBucketLoopMode] = (false,)
+            if bucketMode == True:
```
Comment (Hoikas) [inline Python/tldnWRCCBrain.py:476]: Implicity truthy.

File: Python/tldnWRCCBrain.py [hunk 9]
```diff
@@ -244,13 +244,13 @@ def OnServerInitComplete(self):
                 bucketMode = ageSDL[kStringAgeSDLBucketLoopMode][0]
             except:
                 PtDebugPrint("tldnWRCCBrain.OnServerInitComplete():\tCould not read Bucket Loop Mode SDL, defaulting to: ON")
-                bucketMode = true
+                bucketMode = True
                 ageSDL[kStringAgeSDLBucketLoopMode] = (bucketMode,)
 
-            if bucketMode == true:
-                respBucketLoopMode.run(self.key, state=kOn, fastforward=true)
+            if bucketMode == True:
```
Comment (Hoikas) [inline Python/tldnWRCCBrain.py:250]: Implicity truthy.

File: Python/tldnWRCCBrain.py [hunk 10]
```diff
@@ -146,9 +146,9 @@ def AvatarPage(self, avObj, pageIn, lastOut):
     def Load(self):
         global boolWRCCOperated
         
-        solo = true
+        solo = True
         if len(PtGetPlayerList()):
-            solo = false
+            solo = False
```
Comment (Hoikas) [inline Python/tldnWRCCBrain.py:151]: `solo = bool(PtGetPlayerList())`

File: Python/tldnVaporScope.py [hunk 11]
```diff
@@ -129,9 +129,9 @@ def OnFirstUpdate(self):
 
     def Load(self):
         global boolScopeOperated
-        solo = true
+        solo = True
         if len(PtGetPlayerList()):
-            solo = false
+            solo = False
```
Comment (Hoikas) [inline Python/tldnVaporScope.py:134]: solo = bool(PtGetPlayerList())

File: Python/tldnPwrTwrPeriscope.py [hunk 12]
```diff
@@ -136,9 +136,9 @@ def OnFirstUpdate(self):
 
 
     def Load(self):
-        solo = true
+        solo = True
         if len(PtGetPlayerList()):
-            solo = false
+            solo = False
```
Comment (Hoikas) [inline Python/tldnPwrTwrPeriscope.py:141]: `solo = bool(PtGetPlayerList())`

File: Python/psnlBookshelf.py [hunk 13]
```diff
@@ -198,9 +198,9 @@ def OnServerInitComplete(self):
             actLock.disable()
             actTray.disable()
 
-        solo = true
+        solo = True
         if len(PtGetPlayerList()):
-            solo = false
+            solo = False
```
Comment (Hoikas) [inline Python/psnlBookshelf.py:203]: `solo = bool(PtGetPlayerList())`

File: Python/philBookshelf.py [hunk 14]
```diff
@@ -90,9 +90,9 @@ def OnServerInitComplete(self):
 
 
     def OnNotify(self,state,id,events):
-        boolLinkerIsMe = false
+        boolLinkerIsMe = False
         if PtWasLocallyNotified(self.key):
-            boolLinkerIsMe = true
+            boolLinkerIsMe = True
```
Comment (Hoikas) [inline Python/philBookshelf.py:95]: `boolLinkerIsMe = PtWasLocallyNotified(self.key)`

File: Python/kdshVault.py [hunk 15]
```diff
@@ -178,9 +178,9 @@ def Load(self):
         global ButtonsPushed
         ageSDL = PtGetAgeSDL()     
         
-        solo = true
+        solo = True
         if len(PtGetPlayerList()):
-            solo = false
+            solo = False
```
Comment (Hoikas) [inline Python/kdshVault.py:183]: `solo = bool(PtGetPlayerList())`

File: Python/kdshTreeRings.py [hunk 16]
```diff
@@ -163,9 +163,9 @@ def OnServerInitComplete(self):
         print "/tMiddleRing: ", MiddleRing
         print "/tInnerRing: ", InnerRing
 
-        solo = true
+        solo = True
         if len(PtGetPlayerList()):
-            solo = false
+            solo = False
```
Comment (Hoikas) [inline Python/kdshTreeRings.py:168]: `solo = bool(PtGetPlayerList())`

File: Python/grtzMarkerScopes.py [hunk 17]
```diff
@@ -86,9 +86,9 @@ def OnFirstUpdate(self):
     def Load(self):
         global boolScopeOperated
 
-        solo = true
+        solo = True
         if len(PtGetPlayerList()):
-            solo = false
+            solo = False
```
Comment (Hoikas) [inline Python/grtzMarkerScopes.py:91]: `solo = bool(PtGetPlayerList())`

File: Python/grsnWallPython.py [hunk 18]
```diff
@@ -464,11 +464,11 @@ def OnClimbingWallInit(self,type,state,value):
         
         
         print"grsnClimbingWall::OnClimbingWallInit type ",type," state ",state," value ",value
-        if (ReceiveInit == false):
+        if (ReceiveInit == False):
```
Comment (Hoikas) [inline Python/grsnWallPython.py:467]: `if not ReceiveInit:`

File: Python/grsnWallImagerDisplayS.py [hunk 19]
```diff
@@ -125,11 +125,11 @@ def OnClimbingWallInit(self,type,state,value):
 
     def OnClimbingWallEvent(self,type,state,value):
         
-        if (type == ptClimbingWallMsgType.kAddBlocker and value == false):            #display wall settings
+        if (type == ptClimbingWallMsgType.kAddBlocker and value == False):            #display wall settings
             southWall.value[state].runAttachedResponder(kTeamLightsOn)
             print"Imager display S drawing wall index",state
                     
-        elif (type == ptClimbingWallMsgType.kRemoveBlocker and value == false):
+        elif (type == ptClimbingWallMsgType.kRemoveBlocker and value == False):
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayS.py:132]: `not value`

File: Python/grsnWallImagerDisplayS.py [hunk 20]
```diff
@@ -125,11 +125,11 @@ def OnClimbingWallInit(self,type,state,value):
 
     def OnClimbingWallEvent(self,type,state,value):
         
-        if (type == ptClimbingWallMsgType.kAddBlocker and value == false):            #display wall settings
+        if (type == ptClimbingWallMsgType.kAddBlocker and value == False):            #display wall settings
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayS.py:128]: `not value`

File: Python/grsnWallImagerDisplayS.py [hunk 21]
```diff
@@ -110,11 +110,11 @@ def OnClimbingWallInit(self,type,state,value):
         global ReceiveInit
         
         print"grsnClimbingWall::OnClimbingWallInit type ",type," state ",state," value ",value
-        if (ReceiveInit == false):
+        if (ReceiveInit == False):
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayS.py:113]: `if not ReceiveInit:`

File: Python/grsnWallImagerDisplayN.py [hunk 22]
```diff
@@ -124,11 +124,11 @@ def OnClimbingWallInit(self,type,state,value):
     
     def OnClimbingWallEvent(self,type,state,value):
         
-        if (type == ptClimbingWallMsgType.kAddBlocker and value == true):            #display wall settings
+        if (type == ptClimbingWallMsgType.kAddBlocker and value == True):            #display wall settings
             northWall.value[state].runAttachedResponder(kTeamLightsOn)
             print"Imager display N drawing n wall index",state
                     
-        elif (type == ptClimbingWallMsgType.kRemoveBlocker and value == true):
+        elif (type == ptClimbingWallMsgType.kRemoveBlocker and value == True):
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayN.py:131]: implicit truthy

File: Python/grsnWallImagerDisplayN.py [hunk 23]
```diff
@@ -124,11 +124,11 @@ def OnClimbingWallInit(self,type,state,value):
     
     def OnClimbingWallEvent(self,type,state,value):
         
-        if (type == ptClimbingWallMsgType.kAddBlocker and value == true):            #display wall settings
+        if (type == ptClimbingWallMsgType.kAddBlocker and value == True):            #display wall settings
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayN.py:127]: implicit truthy

File: Python/grsnWallImagerDisplayN.py [hunk 24]
```diff
@@ -109,11 +109,11 @@ def OnClimbingWallInit(self,type,state,value):
         global ReceiveInit
         
         print"grsnClimbingWall::OnClimbingWallInit type ",type," state ",state," value ",value
-        if (ReceiveInit == false):
+        if (ReceiveInit == False):
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayN.py:112]: `if not ReceiveInit:`

File: Python/grsnPowerOn.py [hunk 25]
```diff
@@ -355,11 +355,11 @@ def OnNotify(self,state,id,events):
...
                 print"weight sound looping down"
                 weightStartSoundResp.run(self.key,state='Loop',avatar=triggerer)
-                weightNearDownEnd = false
-                weightSoundLoopingDown = true
+                weightNearDownEnd = False
+                weightSoundLoopingDown = True
                 
-            if (generatorPrimed == false):
-                generatorPrimed = true
+            if (generatorPrimed == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:361]: `if not generatorPrimed:`

File: Python/grsnPowerOn.py [hunk 26]
```diff
@@ -368,9 +368,9 @@ def OnNotify(self,state,id,events):
         if (id == weightUnTrigger.id and generatorPrimed):
             if (weightEngageDisabled):
                 return
-            scaleEngaged = false
-            if (gearOn == false):
-                generatorPrimed = true
+            scaleEngaged = False
+            if (gearOn == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:372]: `if not gearOn:`

File: Python/grsnPowerOn.py [hunk 27]
```diff
@@ -400,33 +400,33 @@ def OnNotify(self,state,id,events):
...
             elif (weightDirection == kUp):
-                weightNearDownEnd = false
-                weightSoundLoopingDown = false
+                weightNearDownEnd = False
+                weightSoundLoopingDown = False
             return
         
             #weightLevel=0
-            #if (brake01On == false):
+            #if (brake01On == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:427]: `if not break01On:`

File: Python/grsnPowerOn.py [hunk 28]
```diff
@@ -400,33 +400,33 @@ def OnNotify(self,state,id,events):
...
             #    weightLevel = weightLevel + 1
-            #if (brake02On == false):
+            #if (brake02On == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:429]: `if not break02On:`

File: Python/clftIntroMusic.py [hunk 29]
```diff
@@ -87,7 +87,7 @@ def __init__(self):
     def OnNotify(self,state,id,events):
         global musicState
         
-        if state == false:
+        if state == False:
```
Comment (Hoikas) [inline Python/clftIntroMusic.py:90]: `if not state:` (state is a float IIRC)

File: Python/GiraBugs.py [hunk 30]
```diff
@@ -217,15 +217,15 @@ def OnBehaviorNotify(self,behavior,id,state):
         global numJumps
         global running
         
-        if (state == false):
+        if (state == False):
```
Comment (Hoikas) [inline Python/GiraBugs.py:220]: `if not state:`

File: Python/GardenBugs.py [hunk 31]
```diff
@@ -303,17 +303,17 @@ def OnBehaviorNotify(self,behavior,id,state):
         global numJumps
         global running
         
-        if (state == false):
+        if (state == False):
```
Comment (Hoikas) [inline Python/GardenBugs.py:306]: `if not state:`

File: Python/grsnPowerOn.py [hunk 32]
```diff
@@ -447,42 +447,42 @@ def OnNotify(self,state,id,events):
...
+                generatorPrimed = False
                 if (gearOn):
                     return
                 flashingLightsResponder.run(self.key,state='Off',avatar=PtGetLocalAvatar())
                 if (mainOn):
                     ageSDL[mainSwitchSDL.value] = (0,)
                     mainSwitchResp.run(self.key,state='PrimerExpired',avatar=triggerer)
                     switchGlowResponder.run(self.key,state='Off',avatar=triggerer)
-                if (brake01On == false):
+                if (brake01On == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:461]: Implicit

File: Python/grsnPowerOn.py [hunk 33]
```diff
@@ -447,42 +447,42 @@ def OnNotify(self,state,id,events):
...
                     gearBrake01Resp.run(self.key,state='AutoReturn',avatar=triggerer)
                     ageSDL[gearBrake01SDL.value] = (1,)
-                if (brake02On == false):
+                if (brake02On == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:464]: Implicit

File: Python/grsnPowerOn.py [hunk 34]
```diff
@@ -447,42 +447,42 @@ def OnNotify(self,state,id,events):
...
                     gearBrake02Resp.run(self.key,state='AutoReturn',avatar=triggerer)
                     ageSDL[gearBrake02SDL.value] = (1,)
                 return
         
         if (id == gearBrake01Resp.id):
             #lock just finished unlocking, check to see that power didn't go off while it was in motion,
-            if (generatorPrimed == false):
+            if (generatorPrimed == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:471]: Implicit

File: Python/grsnPowerOn.py [hunk 35]
```diff
@@ -447,42 +447,42 @@ def OnNotify(self,state,id,events):
...
                 gearBrake01Resp.run(self.key,state='AutoReturn',avatar=triggerer)
                 ageSDL[gearBrake01SDL.value] = (1,)
         
         if (id == gearBrake02Resp.id):
             #lock just finished unlocking, check to see that power didn't go off while it was in motion,
-            if (generatorPrimed == false):
+            if (generatorPrimed == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:477]: Implicit

File: Python/grsnPowerOn.py [hunk 36]
```diff
@@ -492,22 +492,22 @@ def OnNotify(self,state,id,events):
             if (gearStartingDown):
                 print"main power responder callback - gear finished initial descent" 
                 #gear is finished moving partway down, okay to start second half
-                gearStartingDown = false
-                if (gearStopping == false):
+                gearStartingDown = False
+                if (gearStopping == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:496]: Implicit

File: Python/grsnPowerOn.py [hunk 37]
```diff
@@ -492,22 +492,22 @@ def OnNotify(self,state,id,events):
...
                     print"main power responder callback - gear also stopped, begin final descent"
                     mainPowerOffResponder.run(self.key,state='GearDown',avatar=PtGetLocalAvatar())
                 else:
                     print"main power responder callback - gear still turning, wait for it to stop"
                 return
-            if (gearStopping == false and gearStartingDown == false):
+            if (gearStopping == False and gearStartingDown == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:502]: Implicit

File: Python/grsnPowerOn.py [hunk 38]
```diff
@@ -492,22 +492,22 @@ def OnNotify(self,state,id,events):
...
                 print"main power responder callback - reengage locks"
                 #gear is down, reengage the locks
-                if (generatorPrimed == false):
-                    if (brake01On == false):
+                if (generatorPrimed == False):
+                    if (brake01On == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:506]: Implicit

File: Python/grsnPowerOn.py [hunk 39]
```diff
@@ -492,22 +492,22 @@ def OnNotify(self,state,id,events):
...
                         gearBrake01Resp.run(self.key,state='AutoReturn',avatar=triggerer)
                         ageSDL[gearBrake01SDL.value] = (1,)
                     
-                    if (brake02On == false):
+                    if (brake02On == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:510]: Implicit

File: Python/grsnPowerOn.py [hunk 40]
```diff
@@ -537,18 +537,18 @@ def OnNotify(self,state,id,events):
...
                     mainPowerOffResponder.run(self.key,state='GearOff',avatar=triggerer)
-                    gearStopping = true
-                    gearStartingDown = true
+                    gearStopping = True
+                    gearStartingDown = True
                     ageSDL[gearSwitchSDL.value]=(0,)
                     
                 else: # gear not on, but locks may be...
-                    if (brake01On == false):
+                    if (brake01On == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:545]: Implicit

File: Python/grsnPowerOn.py [hunk 41]
```diff
@@ -537,18 +537,18 @@ def OnNotify(self,state,id,events):
...
                         if (gearStartingDown):
                             return
                         gearBrake01Resp.run(self.key,state='AutoReturn',avatar=triggerer)
                         ageSDL[gearBrake01SDL.value] = (1,)
                     
-                    if (brake02On == false):
+                    if (brake02On == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:551]: Implicit

File: Python/grsnPowerOn.py [hunk 42]
```diff
@@ -592,14 +592,14 @@ def OnNotify(self,state,id,events):
                 mainPowerOnResponder.run(self.key,avatar=triggerer)
                 flashingLightsResponder.run(self.key,state='On',avatar=triggerer)
                 gearGlowResponder.run(self.key,state='On',avatar=triggerer)   
-                if (generatorPrimed == false):
+                if (generatorPrimed == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:595]: Implicit

File: Python/grsnPowerOn.py [hunk 43]
```diff
@@ -629,10 +629,10 @@ def OnNotify(self,state,id,events):
                     secondFloorDoors.run(self.key,state='TurnOff',avatar=PtGetLocalAvatar())
                  
                 else:
-                    if (generatorPrimed and brake01On == false and brake02On == false):
+                    if (generatorPrimed and brake01On == False and brake02On == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:632]: Implicit

File: Python/grsnPowerOn.py [hunk 44]
```diff
@@ -694,7 +694,7 @@ def OnNotify(self,state,id,events):
             if (gearOn):
                 gearBrake01Resp.run(self.key,state='Trip',avatar=triggerer)
                 return
-            elif (mainOn == false):
+            elif (mainOn == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:697]: Implicit

File: Python/grsnPowerOn.py [hunk 45]
```diff
@@ -718,7 +718,7 @@ def OnNotify(self,state,id,events):
             if (gearOn):
                 gearBrake02Resp.run(self.key,state='Trip',avatar=triggerer)
                 return
-            elif (mainOn == false):
+            elif (mainOn == False):
```
Comment (Hoikas) [inline Python/grsnPowerOn.py:721]: Implicit

File: Python/xTelescope.py (hunk 1 above)
Comment (aqua-uru) [reply Python/xTelescope.py:94]: `solo = not PtGetPlayerList()`, correct?

Comment (Hoikas) [reply Python/xTelescope.py:94]: Yes, sorry about that 😢

**Comment by aqua-uru** [comment]
Aaah no I just noticed I did some accidental automatic import sorting in those last commits... Too late now, will fix tomorrow.

**Comment by aqua-uru** [comment]
Phew, fixed!

**Comment by aqua-uru** [comment]
Some more semi-relevant prettifying...

File: Python/grsnWallImagerDisplayS.py [hunk 46]
```diff
@@ -110,31 +108,31 @@ def OnClimbingWallInit(self,type,state,value):
...
 
     def OnClimbingWallEvent(self,type,state,value):
         
-        if (type == ptClimbingWallMsgType.kAddBlocker and value == false):            #display wall settings
+        if (type == ptClimbingWallMsgType.kAddBlocker and not value:            #display wall settings
             southWall.value[state].runAttachedResponder(kTeamLightsOn)
             print"Imager display S drawing wall index",state
                     
-        elif (type == ptClimbingWallMsgType.kRemoveBlocker and value == false):
+        elif (type == ptClimbingWallMsgType.kRemoveBlocker and not value:
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayS.py:130]: Syntax Error

File: Python/grsnWallImagerDisplayS.py [hunk 47]
```diff
@@ -110,31 +108,31 @@ def OnClimbingWallInit(self,type,state,value):
...
             print "begin receiving total game state"
         
-        elif (type == ptClimbingWallMsgType.kAddBlocker and state > 0 and value == 0):
+        elif type == ptClimbingWallMsgType.kAddBlocker and state > 0 and value == 0:
             southWall.value[state].runAttachedResponder(kTeamLightsOn)
 
     def OnClimbingWallEvent(self,type,state,value):
         
-        if (type == ptClimbingWallMsgType.kAddBlocker and value == false):            #display wall settings
+        if (type == ptClimbingWallMsgType.kAddBlocker and not value:            #display wall settings
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayS.py:126]: Syntax Error

File: Python/grsnWallImagerDisplayS.py [hunk 48]
```diff
@@ -97,10 +97,8 @@ def OnServerInitComplete(self):
         global ReceiveInit
         
         PtDebugPrint("grsnWallPython::OnServerInitComplete")        
-        solo = true
-        if len(PtGetPlayerList()):
-            solo = false
-            ReceiveInit = true
+        solo = not PtGetPlayerList()
+            ReceiveInit = True
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayS.py:101]: Syntax Error

File: Python/grsnWallPython.py [hunk 49]
```diff
@@ -570,17 +570,15 @@ def OnServerInitComplete(self):
         global ReceiveInit
         
         PtDebugPrint("grsnWallPython::OnServerInitComplete")        
-        solo = true
-        if len(PtGetPlayerList()):
-            solo = false
-            ReceiveInit = true
+        solo = not PtGetPlayerList()
+            ReceiveInit = True
```
Comment (Hoikas) [inline Python/grsnWallPython.py:574]: Syntax Error

File: Python/grsnWallImagerDisplayN.py [hunk 50]
```diff
@@ -96,10 +96,8 @@ def OnServerInitComplete(self):
         global ReceiveInit
         
         PtDebugPrint("grsnWallPython::OnServerInitComplete")        
-        solo = true
-        if len(PtGetPlayerList()):
-            solo = false
-            ReceiveInit = true
+        solo = not PtGetPlayerList()
+            ReceiveInit = True
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayN.py:100]: Syntax Error

File: Python/grsnWallImagerDisplayN.py [hunk 51]
```diff
@@ -109,31 +107,31 @@ def OnClimbingWallInit(self,type,state,value):
...
             print "begin receiving total game state"
         
-        elif (type == ptClimbingWallMsgType.kAddBlocker and state > 0 and value):
+        elif type == ptClimbingWallMsgType.kAddBlocker and state > 0 and value:
             northWall.value[state].runAttachedResponder(kTeamLightsOn)
     
     def OnClimbingWallEvent(self,type,state,value):
         
-        if (type == ptClimbingWallMsgType.kAddBlocker and value == true):            #display wall settings
+        if (type == ptClimbingWallMsgType.kAddBlocker and value:            #display wall settings
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayN.py:125]: Syntax Error

File: Python/grsnWallImagerDisplayN.py [hunk 52]
```diff
@@ -109,31 +107,31 @@ def OnClimbingWallInit(self,type,state,value):
...
             northWall.value[state].runAttachedResponder(kTeamLightsOn)
             print"Imager display N drawing n wall index",state
                     
-        elif (type == ptClimbingWallMsgType.kRemoveBlocker and value == true):
+        elif (type == ptClimbingWallMsgType.kRemoveBlocker and value:
```
Comment (Hoikas) [inline Python/grsnWallImagerDisplayN.py:129]: Syntax Error

File: Python/ahnyPressurePlates.py [hunk 53]
```diff
@@ -183,20 +183,20 @@ def OnNotify(self,state,id,events):
...
                                     occupiedZones[index] = occupiedZones[index] -1
                                 if Sphere.value == "Sphere02" and index in treeToZoneKey:
-                                    if ((not ageSDL[SDLTrees.value][treeToZoneKey.index(index)]) and( occupiedZones[index] == 0)):
+                                    if (not ageSDL[SDLTrees.value][treeToZoneKey.index(index)]) and( occupiedZones[index] == 0):
                                         if respLightList != []:
                                             respClockLights.run(self.key, state='off', objectName=respLightList[index], netForce=1)
                                 else:
 
-                                    if (respLightList != []) and (occupiedZones[index] == 0):# 
+                                    if respLightList != []) and (occupiedZones[index] == 0:#
```
[the remark this answers is not in the document]
Comment (aqua-uru) [reply Python/ahnyPressurePlates.py]: Should I fix this here or leave it for later as a stylistic change?

**Pull request merged**